radio/src/thirdparty/FreeRTOS/include/FreeRTOS.h

```diff
@@ -175,7 +175,7 @@
 #endif
 
 #ifndef INCLUDE_uxTaskGetStackHighWaterMark
-    #define INCLUDE_uxTaskGetStackHighWaterMark    0
+    #define INCLUDE_uxTaskGetStackHighWaterMark    1
```

Comment:
By FreeRTOS convention
#define INCLUDE_uxTaskGetStackHighWaterMark  1
should be defined in https://github.com/opentx/opentx/blob/3djc/FreeRTOS/radio/src/FreeRTOSConfig.h
and here under thirdparty\FreeRTOS the setting should be left unchanged.